e2e-tests/create-mastra/create-mastra.test.ts

```
@@ -1,171 +1,254 @@
+      const agentsResponse = await waitForServer(`http://localhost:${port}/api/agents`);
+      const agents = (await agentsResponse.json()) as Record<string, { name?: string }>;
+      expect(agents.agent?.name).toBe('Agent');
+    } finally {
+      if (server.exitCode === null && server.signalCode === null && server.pid) {
+        if (process.platform === 'win32') {
+          server.kill('SIGTERM');
+        } else {
+          try {
+            process.kill(-server.pid, 'SIGTERM');
+          } catch (error) {
+            if ((error as { code?: string }).code !== 'ESRCH') throw error;
```

Biome (2.5.3)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)